src/codelabs/null-safety.md

```diff
@@ -384,8 +388,8 @@ void main() {
 
 ### Exercise: Null checking
 
-In the code below, add an `if` statement to the beginning of `getLength` that
-returns zero if `str` is null:
+In the following code, add an `if` statement to the beginning of
+`getLength` that returns zero if `str` has no value:
```

Comment:
I feel "is null" is better than "no value" because "no value" could also be taken to mean it hasn't been assigned a value yet.

```suggestion
`getLength` that returns zero if `str` is `null`:
```